src/lib/redux/userData.ts

```diff
@@ -49,31 +67,35 @@ const checkOnline = (): void => {
         //refresh data code here
         refreshDataBackground();
       }
-      store.dispatch(setIsOnline({ isOnline: true }))
-
-    }
-    ,
+      store.dispatch(setIsOnline({ isOnline: true }));
+    },
     error: () => store.dispatch(setIsOnline({ isOnline: false })),
   });
 };
 
-const getUserId = (): string => {
-  const state = store.getState();
-  let userId = '';
+export const selectCurrentUserId = (state: RootState): string => state.userData.currentUserId;
 
-  if (state.userData && state.userData.user) {
-    userId = state.userData.user.id;
-  }
+export const selectUserIsSignedIn = createSelector(selectCurrentUserId, (currentUserId) => currentUserId !== '');
```

Comment:
NIT: maybe `selectCurrentUserIsSignedIn`? Since we're using `CurrentUser` for other naming